### DISCUSSION TITLE
How to update heidisql to latest version?

### DISCUSSION THREAD

**Discussion by kstingel** [leokhoa/laragon#676] [Q&A] [answered]
default installed version of HeidiSQL is 12.1.0.6537
latest (portable) version is 12.6.0.6725
what is the easiest way to update?

**Comment by leokhoa** [comment, marked as answer]
@kstingel 
Rename the previous one to `bin\heidisql.bak 
Then, just download the Portable in extract to `bin\heidisql`
https://www.heidisql.com/downloads/releases/HeidiSQL_12.6_64_Portable.zip

**Reply by kstingel** [reply]
@leokhoa thank-you
